pydantic_ai_slim/pydantic_ai/durable_exec/temporal/_run_context.py

```diff
@@ -42,6 +42,7 @@ def __getattribute__(self, name: str) -> Any:
     def serialize_run_context(cls, ctx: RunContext[Any]) -> dict[str, Any]:
         """Serialize the run context to a `dict[str, Any]`."""
         return {
+            'run_id': ctx.run_id,
```

Comment:
Added it as a part of temporal serialization, and I added a test for the round trip as well.